source/_integrations/matter.markdown

```diff
@@ -180,6 +179,22 @@ This guide describes how to add a new device. This will use the Bluetooth connec
 
 <lite-youtube videoid="Fk0n0r0eKcE" videotitle="Add Matter device via Android app in Home Assistant"></lite-youtube>
 
+### Troubleshooting the installation
+
+Check these steps if you are experiencing issues when trying to add a Matter device using the Home Assistant Companion app on your Android phone.
+
+#### Symptom
+
+While trying to add the Matter device, I get an error stating that *Matter is currently unavailable*. 
+
+#### Remedy
+
+This could mean that not all required Matter modules that are needed by the Home Assistant Companion App have been downloaded yet. Try the following steps:
+
+1. Wait up to 24 hours for the Google Play services to download the necessary Matter modules. For more information, refer to this [Troubleshooting Guide from Google](https://developers.home.google.com/matter/verify-services).
+2. If this did not work, try reinstalling the Home Assistant Companion app.
```

Comment:
1. Wait up to 24 hours for the Google Play services to download the necessary Matter modules.
2. If this did not work, try reinstalling the Home Assistant Companion app.
3. Try installing the Google Home app. Technically this is not required, but it might trigger another installation attempt of the Matter modules.
4. Refer to this [Troubleshooting Guide from Google](https://developers.home.google.com/matter/verify-services).